include/deal.II/lac/sparse_vanka.templates.h

```diff
@@ -133,39 +133,34 @@ SparseVanka<number>::compute_inverses()
   // consecutive, with other
   // consecutive regions where we
   // do not have to do something
-  std::vector<std::pair<size_type, unsigned int>> blocking(n_threads);
+  std::vector<std::pair<size_type, unsigned int>> blocking(n_tasks);
 
   unsigned int c      = 0;
-  unsigned int thread = 0;
+  unsigned int task_n = 0;
   blocking[0].first   = 0;
 
-  for (size_type i = 0; (i < matrix->m()) && (thread + 1 < n_threads); ++i)
+  for (size_type i = 0; (i < matrix->m()) && (task_n + 1 < n_tasks); ++i)
```

Comment:
Is this loop doing something that is expressible through the `Threads::split_range()` function?

Reply:
I believe the answer is no, since we don't apply the smoother to all dofs - the range is chopped up so that we have an even number of smoothed dofs per range.

Reply:
OK.